configuration/configure-unhealthy-cluster-removal.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+[id="configure-unhealthy-cluster-removal"]
+= Configuring automatic removal of unhealthy clusters
+include::modules/common-attributes.adoc[]
+:context: configure-unhealthy-cluster-removal
+
+toc::[]
+
+[role="_abstract"]
+{product-title} ({product-title-short}) provides the ability to automatically remove unhealthy clusters in your system. If this feature is enabled, when Central has been unable to reach Sensor in a cluster for a configured period of time, the cluster is considered unhealthy and is deleted (decommissioned). Automatically deleting unhealthy clusters allows you to more easily monitor only active clusters. You can configure cluster removal settings by using the {product-title-short} portal or the application programming interface (API).
+
+This setting is disabled by default. To enable this setting, enter a nonzero number in the *Decommissioned cluster age* field, as described in the following procedure. The *Decommissioned cluster age* field indicates the number of days that a cluster can remain unreachable before it is considered unhealthy. When a cluster is unhealthy, the Clusters page displays the status of the cluster and the number of days after which the cluster will be deleted if it continues to remain unhealthy. After a cluster is removed, the removal is documented with an `info` log in the Central logs.
```

Comment:
The terms 'removed', 'deleted', and 'decommissioned' are used interchangeably throughout the topic. Is this intentional? If not, consider using a consistent term throughout.

Reply:
It's referred to as "deleted" and "decommissioned" in the GUI, but I'll try to standardize, good point!

Reply:
OK, I checked and "decommissioned" = unhealthy, which are then deleted, so I will be consistent with using these 2 terms and get rid of "removed."